sites/platform/static/files/fetch/examples/relationships/kafka

```diff
@@ -2,8 +2,11 @@
     "service": "kafka25",
     "ip": "169.254.27.10",
     "hostname": "t7lv3t3ttyh3vyrzgqguj5upwy.kafka25.service._.eu-3.platformsh.site",
-    "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "kafka.internal",
+    "cluster": "rjify4yjcwxaa-master-7rqtwti",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.kafka25.service._.eu-1.platformsh.site",